extensions/amp-story/1.0/amp-story-embedded-component.js

```diff
@@ -405,6 +409,7 @@ export class AmpStoryEmbeddedComponent {
         this.onFocusedStateUpdate_(null);
         this.scheduleEmbedToPause_(component.element);
         this.toggleExpandedView_(component.element);
+        this.historyService_.push(() => this.close_());
```

Comment:
You should consider passing the current `historyState` as a second parameter, otherwise you'll lose all the context on page reload (eg: it'd go back to the first page of the story).
Example: https://github.com/ampproject/amphtml/blob/master/extensions/amp-story/1.0/amp-story-page-attachment.js#L149

Reply:
So the context isn't lost, since it was already pushed to the service before. Specifically, the page will still load in the same active page when reloading. But I misunderstood your question and thought you wanted to persist the state of the expanded component after reloading, as right now it will be closed when reloading the page.

In order to fix this, we would also have to skip the animation of opening the embed into its expanded mode. Since this animation follows a complicated code path (including a debounced function that prepares the animation), I think it deserves its own PR. WDYT?

Reply:
Got it, I didn't know that the `historyService` automatically propagated the current history state to the new history entry. https://github.com/ampproject/amphtml/blob/master/src/service/history-impl.js#L524